packages/ai-jsx/src/lib/anthropic.tsx

```diff
@@ -0,0 +1,178 @@
+import Anthropic from '@anthropic-ai/sdk';
+import { getEnvVar } from './util.js';
+import * as AI from '../index.js';
+import { Node } from '../index.js';
+import { ChatOrCompletionModelOrBoth } from './model.js';
+import {
+  AssistantMessage,
+  ChatProvider,
+  FunctionCall,
+  FunctionResponse,
+  ModelProps,
+  SystemMessage,
+  UserMessage,
+  ModelPropsWithChildren,
+} from '../core/completion.js';
+import { AIJSXError, ErrorCode } from '../core/errors.js';
+
+export const anthropicClientContext = AI.createContext<Anthropic>(
+  new Anthropic({
+    apiKey: getEnvVar('ANTHROPIC_API_KEY', false),
+  })
+);
+
+type ValidCompletionModel = never;
+/**
+ * The set of valid Claude models.
+ *
+ * @see https://docs.anthropic.com/claude/reference/complete_post.
+ */
+type ValidChatModel =
+  | 'claude-1'
+  | 'claude-1-100k'
+  | 'claude-instant-1'
+  | 'claude-instant-1-100k'
+  | 'claude-1.3'
+  | 'claude-1.3-100k'
+  | 'claude-1.2'
+  | 'claude-1.0'
+  | 'claude-instant-1.1'
+  | 'claude-instant-1.1-100k'
+  | 'claude-instant-1.0';
+
+type AnthropicModelChoices = ChatOrCompletionModelOrBoth<ValidChatModel, ValidCompletionModel>;
+
+/**
+ * An AI.JSX component that invokes an Anthropic Large Language Model.
+ * @param children The children to render.
+ * @param chatModel The chat model to use.
+ * @param completionModel The completion model to use.
+ * @param client The OpenAI client.
+ */
+export function OpenAI({
```

Comment:
Is this function called the right thing?

Reply:
NOPE. Good catch.